### ISSUE TITLE
Machine God Chaplain cannot convert self.

### ISSUE THREAD

**Issue by silveredsoul** [tgstation/tgstation#63100]
## Round ID:

173762

## Testmerges:

Don't know the exact testmerge, but in the changelog I saw this when looking for what the issue was 
"tralezab changed: fixes non chaplains being able to invoke rites."  on November 25th, 2021.

## Reproduction:

Basically, at least one rite, "Synthetic Conversion" requires someone to use the ritual on the chaplain so the chaplain can convert. Why would a chaplain of a machine god, not convert, or be able to convert? There are rarely multiple chaplains except on cult rounds perhaps with an attentive HoP. At this point, thanks to this "Fix" the chaplain can't be converted into a android, so why pick it? I only really play this and the fire one, so I'm not sure how other rituals are handled to affect the chaplain, but not being able to get the main benefit of a sect as chaplain seems pointless. 

TLDR Chaplain needs another to perform the rite on him to get benefits, but only chaplains can perform rites now, and it's impossible? to do on one self due to being bound/buckled/laid down on the altar. No one else is "holy enough" to perform rite. Sad Chaplain cannot be rid of flesh.

1. Be Chaplain
2. Be Machine Sect
3.  Get 1000 Favor
4. Be unable to convert self due to being unable to do a altar rite while bound to the altar
5. Everyone else is "Not holy enough" to perform the rite
6. Cry as the omnissiah has forsaken you.
